client/app/cloud/project/compute/infrastructure/list/cloud-project-compute-infrastructure-list.html

```diff
@@ -3,89 +3,87 @@
-                    </li>
-                    <li class="divider"
-                        data-ng-if="ComputeInfrastructureListCtrl.TARGET === 'US'">
-                    </li>
-                    <li data-ng-if="ComputeInfrastructureListCtrl.TARGET === 'US'">
-                        <button type="button"
-                                class="btn fs14"
-                                data-ng-click="ComputeInfrastructureListCtrl.InfrastructureService.orderCredit()"
-                                data-translate="cpb_vouchers_add_credit_title">
-                        </button>
-                    </li>
-                    <li class="divider" role="presentation"></li>
-                    <li role="menuitem">
-                        <button type="button" class="btn fs14"
-                                at-internet-click="{ name : 'cloud-project::cloud-project-compute::cloud-project-compute-infrastructure-delete-project' }"
-                                data-ng-bind=":: 'cpb_details_project_delete' | translate"
-                                data-ng-click="ComputeInfrastructureListCtrl.InfrastructureService.openDeleteProjectModal()"
-                                write-right-required>
-                        </button>
-                    </li>
-                </ul>
-            </div>
-        </div>
-        <div class="col-xs-9 text-right">
-            <div class="d-inline-block">
-                <span class="oui-label align-middle d-inline-block mb-0 mr-3"
-                      data-translate="cpci_menu_view_label"></span>
-                <nav class="oui-button-group align-middle mb-0">
-                    <a class="oui-button oui-button_secondary"
-                       ui-sref="iaas.pci-project.compute.infrastructure.diagram()"
-                       data-translate="cpci_menu_view_diagram"></a>
-                    <a class="oui-button oui-button_primary"
-                       ui-sref="iaas.pci-project.compute.infrastructure.list()"
-                       data-translate="cpci_menu_view_list"></a>
-                </nav>
-            </div>
-        </div>
+    <div class="col-xs-12 mb-5 text-right d-inline-block">
+        <span class="oui-label align-middle d-inline-block mb-0 mr-3"
+              data-translate="cpci_menu_view_label"></span>
+        <nav class="oui-button-group align-middle mb-0">
+            <a class="oui-button oui-button_secondary"
+               data-ui-sref="iaas.pci-project.compute.infrastructure.diagram()"
+               data-translate="cpci_menu_view_diagram"></a>
+            <a class="oui-button oui-button_primary"
+               data-ui-sref="iaas.pci-project.compute.infrastructure.list()"
+               data-translate="cpci_menu_view_list"></a>
+        </nav>
     </div>
 
     <section class="row">
         <oui-datagrid class="col-xs-12" data-rows="ComputeInfrastructureListCtrl.table.items">
+            <extra-top>
+                <div class="col-xs-3 pl-0">
```

Comment:
⚠️ [Bootlint E014](https://github.com/twbs/bootlint/wiki/E014)

Having a `.col-xs-3` class inside the `<extra-top>` component doesn't make sense in my point of view.

According the to documentation, [`<extra-top>`](https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/datagrid#extra-top-content) is where we use a `<oui-action-menu>` for instance.

Hope it could help you.

Reply:
Have revamped it with `action-menu`. Please have a look now.